config/install/pathauto.pattern.stanford_publications.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: stanford_publications
+label: Publications
+type: 'canonical_entities:node'
+pattern: '/publications/[node:su_publication_topics:0:entity:name]/[node:title]'
```

Comment:
`[node:su_publication_topics:0:entity:name]` token is a non-required field.

Reply:
See the comment above.
If i create a node without the field populated, it produces `/publications/[node:title]`. path auto has gotten smarter about empty tokens.